src/js/@WebParts/ProgramExperienceLog/index.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from "react";
+import IProgramExperienceLogProps, { ProgramExperienceLogDefaultProps } from "./ProgramExperienceLogProps";
+import IProgramExperienceLogState from "./IProgramExperienceLogState";
+import { ExperienceLog } from "prosjektportalen/lib/WebParts";
+import { IDataSourceSearchCustom } from "prosjektportalen/lib/WebParts/DataSource";
+import * as common from "../../@Common";
+import { MessageBar, MessageBarType } from "office-ui-fabric-react/lib/MessageBar";
+import NoStoredProjectsMessage from "../@Components/NoStoredProjectsMessage";
+
+export default class ProgramExperienceLog extends React.Component<IProgramExperienceLogProps, IProgramExperienceLogState> {
+    public static defaultProps = ProgramExperienceLogDefaultProps;
+
+    constructor(props: IProgramExperienceLogProps) {
+        super(props);
+
+        this.state = { isLoading: true };
+    }
+
+    public async componentDidMount() {
+        try {
+            const searchSettings = await this.buildSearchSettingsFromStoredProjects();
+            this.setState({ searchSettings, isLoading: false });
+        } catch (errorMessage) {
+            this.setState({ errorMessage, isLoading: false });
+        }
+    }
+
+    public render(): React.ReactElement<IProgramExperienceLogProps> {
+        if (this.state.errorMessage) {
+            return (
+                <>
+                    <h1>Erfaringslogg</h1>
+                    <MessageBar messageBarType={MessageBarType.error}>{this.state.errorMessage}</MessageBar>
+                </>
+            );
+        }
+        if (this.state.searchSettings === null) {
+            return (
+                <>
+                    <h1>Erfaringslogg</h1>
+                    <NoStoredProjectsMessage />
+                </>
+            );
+        }
+        return (
+            <>

```

Comment:
Hva er <>? Mener du å bruke div?

Reply:
Shorthand for `Fragment` i React. Brukt for å unngå div

Reply:
Hvorfor unngå div? Vi bruker div ellers i løsningen.

Reply:
Hvis det er en god grunn, så hvorfor ikke.

Reply:
Tanken var å unngå unødvendige DOM elementer. Typisk wrappere som bare er der fordi React "krever" det

Reply:
Ok